lib/licensed/sources/mix.rb

```diff
@@ -0,0 +1,396 @@
+# frozen_string_literal: true
+require "strscan"
+
+module Licensed
+  module Sources
+    class Mix < Source
+
+      LOCKFILE = "mix.lock"
+
+      # Returns whether a mix.lock is present
+      def enabled?
+        File.exists?(LOCKFILE)
+      end
+
+      def enumerate_dependencies
+        parsed_dependencies
+      end
+
+      private
+
+      # Returns a memoized Array of Dependency instances.
+      def parsed_dependencies
+        @parsed_dependencies ||= find_packages.map do |name, lock_info|
+          convert_package_to_dependency(name, lock_info)
+        end
+      end
+
+      # Returns the parsed mix.lock information as a Hash.
+      def find_packages
+        lockfile = File.read(LOCKFILE)
```

Comment:
nit: same here, mind using an absolute path?
```suggestion
        lockfile = File.read(config.pwd.join(LOCKFILE))
```